src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: rgb(255, 255, 255);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 100%;
+  flex-grow: 1;
+}
+
+.b1 {
+  background-color: rgb(255,0,0);
+  order: 1;
+}
+
+.b2 {
+  background-color: rgb(204,0,0);
+  order: 2;
+}
+
+.b3 {
+  background-color: rgb(153,0,0);
+  order: 3;
+}
+
+.b4 {
+  background-color: rgb(102,0,0);
+  order: 4;
+}
+
+.b5 {
+  background-color: rgb(51,0,0);
+  order: 5;
+}
+
+.b6 {
+  background-color: rgb(0,0,0);
+  order: 6;
+}
```

Comment:
the order here is redundant. this is the default behavior. try to learn writing less code